layers/+tools/lsp/funcs.el

```diff
@@ -30,3 +30,8 @@
                       :background (face-attribute 'highlight :background nil t)
                       :foreground (face-attribute 'default :foreground nil t))
   )
+
+(defun spacemacs//set-lsp-key-bindings (mode)
+  "Set the key bindings for tern and the given MODE."
```

Comment:
keybindings for lsp

Reply:
I fixed it, then I renamed the function as it does not set the key bindings but the jump handler, see commit:  https://github.com/syl20bnr/spacemacs/commit/7596a156a